### DISCUSSION TITLE
Monika not accepting gifts

### DISCUSSION THREAD

**Discussion by PongMaster69** [Monika-After-Story/MonikaModDev#8395] [QA - tech support + general]
Every gift she receives is illegible to her (I.E. the message you get when you do something wrong). For context I downloaded the new sprite pack and moved all the contents into the folder, and I bulk gifted all of the outfits and I cannot get any of them to be read. I made sure the mod assets when into game folder and that the gift files were removed from their original folders beforehand.

**Comment by ThePotatoGuy** [comment]
Follow the steps [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation)
